components/content/ProseAddress.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+const css = useCssModule();
+</script>
+
+<template>
+  <address :class="css.address">
+    <ContentSlot :use="$slots.default" unwrap="p" />
```

Comment:
nitpick: can we use `useSlots()` instead

Reply:
Okay, I'll check it out.